modules/statics/src/coins.ts

```diff
@@ -304,7 +304,7 @@ export const coins = CoinMap.fromCoins([
   erc20('npxs', 'Pundi X', 18, '0xa15c7ebe1f07caf6bff097d8a589fb8ac49ae5b3', UnderlyingAsset.NPXS),
   erc20('nu', 'NuCypher', 18, '0x4fe83213d56308330ec302a8bd641f1d0113a4cc', UnderlyingAsset.NU),
   erc20('nzdx', 'eToro New Zealand Dollar', 18, '0x6871799a4866bb9068b36b7a9bb93475ac77ac5d', UnderlyingAsset.NZDX),
-  erc20('ocean', 'Ocean Token', 18, '0x7afebbb46fdb47ed17b22ed075cde2447694fb9e', UnderlyingAsset.OCEAN),
+  erc20('ocean', 'Ocean Token', 18, '0x967da4048cD07aB37855c090aAF366e4ce1b9F48', UnderlyingAsset.OCEAN),
```

Comment:
hi ocean!

this is a good one-liner but the ethereum casing needs to be lower cased

thanks

Reply:
thanks, force-pushed to keep the one-liner PR in one commit :-)